python/test/t_SmoothedUniformFactory_std.py

```diff
@@ -0,0 +1,27 @@
+#! /usr/bin/env python
+
+import openturns as ot
+import openturns.experimental as otexp
+import openturns.testing as ott
+
+ot.TESTPREAMBLE()
+
+distribution = ot.SmoothedUniform(-0.5, 1.5, 0.5)
+size = 10000
+sample = distribution.getSample(size)
+factory = otexp.SmoothedUniformFactory()
+estimatedDistribution = factory.build(sample)
+ott.assert_almost_equal(estimatedDistribution.getParameter(), distribution.getParameter(), 1e-2, 1e-2)
+print("distribution=", repr(distribution))
+print("Estimated distribution=", repr(estimatedDistribution))
+estimatedDistribution = factory.build()
+print("Default distribution=", estimatedDistribution)
+estimatedDistribution = factory.build(distribution.getParameter())
+print("Distribution from parameters=", estimatedDistribution)
+estimatedSmoothedUniform = factory.buildAsSmoothedUniform(sample)
+print("SmoothedUniform          =", distribution)
+print("Estimated uniform=", estimatedSmoothedUniform)
+estimatedSmoothedUniform = factory.buildAsSmoothedUniform()
+print("Default uniform=", estimatedSmoothedUniform)
+estimatedSmoothedUniform = factory.buildAsSmoothedUniform(distribution.getParameter())
```

Comment:
This is not actually tested due to the IGNOREOUT. Or is it just to see if the code doesn't crash?

Reply:
yes, I could add some asserts